### PR TITLE
Milestone: Update the database table

### PR DESCRIPTION
**Pull request by Lancelot-SO**
### The following updates were made: 

> Updated the readme file. 

> Added a column species of type string to the animals table. 

> Inserted the following data:

- Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
- Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
- Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
- Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
- Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
- Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.

> Updated the animals table by setting the species column to unspecified
Screenshot 2023-03-28 143024

> Verified that the change was made. Then rolled back the change and verified that the species columns went back to the state before the transaction. 
Screenshot 2023-03-28 143214

> Wrote these queries inside the transaction:

- Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Screenshot 2023-03-28 143745

- Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
Screenshot 2023-03-28 151344

- Commit the transaction.
Screenshot 2023-03-28 151526
- Verify that change was made and persists after commit.

> deleted all records in the animals table, then rolled back the transaction
Screenshot 2023-03-28 151728
> Rolled back the transaction
Screenshot 2023-03-28 151816

>Wrote these queries inside the transaction:

- Deleted all animals born after Jan 1st, 2022.
Screenshot 2023-03-28 152016

- Created a savepoint for the transaction.
Screenshot 2023-03-28 152152

- Update all animals' weight to be their weight multiplied by -1.
Screenshot 2023-03-28 152554

- Rollback to the savepoint
Screenshot 2023-03-28 152651

- Updated all animals' weights that are negative to be their weight multiplied by -1.
Screenshot 2023-03-28 152853

- Commit transaction
Screenshot 2023-03-28 152947

> Wrote queries to answer the following questions:

- How many animals are there?
Screenshot 2023-03-28 153945

- How many animals have never tried to escape?
Screenshot 2023-03-28 154023

- What is the average weight of animals?
Screenshot 2023-03-28 154140

- Who escapes the most, neutered or not neutered animals?
Screenshot 2023-03-28 154248

- What is the minimum and maximum weight of each type of animal?
Screenshot 2023-03-28 154336

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screenshot 2023-03-28 154447

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ After you're finished please remove all the comments and instructions!
 - Vet clinic database
 - animals table
 - Querying data
+- Species column
```
Comment (rotimiazeez) [inline README.md:76]: - Please note that the MIT Lisence link provided is invalid. 😔 Kindly provide a valid license link 👍🏼

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Lancelot-SO 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Lancelot-SO , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] As requested by previous reviewer that the MIT License link provided in README File is invalid.  Kindly provide a valid license link 👍🏼.

* You have updated the Licence file but forgot to add this in the readme file
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Copyright 2021, [ Felix Adjei Sowah ]
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
+
+THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```
Comment (AnselemOdims) [inline MIT.md:8]: - [ ] [OPTIONAL] As pointed out by the previous reviewers, please kindly include this license link in your README `License` section. 
`[MIT](./MIT.md)`

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Lancelot-SO 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**